src/main/java/com/github/psambit9791/jdsp/filter/adaptive/LMSFilter.java

```diff
@@ -0,0 +1,178 @@
+package com.github.psambit9791.jdsp.filter.adaptive;
+
+import java.util.Arrays;
+
+/**
+ * <h1>Least-mean-squares (LMS) adaptive filter</h1>
+ * The LMS adaptive filter is a filter that adapts its filter weights to get an input signal x to match a desired output
+ * signal (= the output of the filter). It does this by trying to minimize the squared error between the desired signal
+ * and the filter output signal.
+ *
+ * @author SiboVanGool
```

Comment:
Keep the name consistent between files. I think I saw "Sibo Van Gool" (with spaces) in another file.